protos/sources/vega/markets.proto

```diff
@@ -275,6 +275,9 @@ message Market {
   MarketTimestamps market_timestamps = 10;
   // The number of decimal places for a position
   int64 position_decimal_places = 11;
+  // Percentage move up and down from the mid price which specifies the range of 
+  // price levels over which automated liquidity provision orders will be deployed
+  double lp_price_range = 12;
```

Comment:
this should be string, we shouldn't use doubles/floats.

Reply:
Ok, thanks. So any legacy doubles (say `LiquidityMonitoringParameters.triggering_ratio`) will get changed to strings in the future?

Reply:
I think those are less important but inside the market nothing should be done with floats.

Reply:
Yes, of course, that's why I've set the `LPPriceRange` field in [core/types/market.go](https://github.com/vegaprotocol/vega/blob/630acafa1b7738c19d3eda58c3b7104cb68648fe/core/types/market.go#L633) to be a `Decimal` from the very beginning, but you're commenting on `markets.proto` which has plenty of doubles. 

Anyway, I've changed it to a string, just trying to understand what the convention should be since I just followed the same approach as was already used for other similar parameters  (e.g. `triggering_ratio`, `scaling_factor` both of which are used inside the market).